build.sh

```diff
@@ -1,3 +1,31 @@
 #!/usr/bin/env bash
 
-packer build -parallel=false hipster.pkr.hcl
\ No newline at end of file
+set -ex
+
+BUILD_VERSION=$(date +%Y%m%d)
+
```

Comment:
Could we use some kind of option to select if only VirtualBox or qemu should be build. Maybe a Makefile would be a more suitable option?

Reply:
Actually a Makefile is a nice idea for that situation. I'll have a look into while crafting the next release (around next week) and add it.

Reply:
Added a makefile :)